django/utils/dateparse.py

```diff
@@ -114,7 +126,8 @@ def parse_duration(value):
 
     The preferred format for durations in Django is '%d %H:%M:%S.%f'.
 
-    Also supports ISO 8601 representation.
+    Also supports ISO 8601 representation,
```

Comment:
Only the Day-Time interval format is supported, as the Year-Month and Mixed interval types cannot be represented as a `datetime.timedelta()` object.